src/scripts/release_picard.sh

```diff
@@ -184,12 +184,12 @@ cd $TMPDIR
 # clone
 git clone $PICARDGITROOT picard
 cd picard
-ant clone-htsjdk
+ant -buildfile build-from-src.xml clone-htsjdk
```

Comment:
Ah now I'm remembering why. This is for the github release.. which releases both picard and htsjdk simultaneously. So if we didn't grab the htsjdk master we would be releasing the previous version of htsjdk. Really these two releases should be decoupled even though they are released at the same time.